Jenkinsfile

```diff
@@ -52,28 +52,21 @@ pipeline {
                 }
             }
 
-        stage('Build .whl & .deb') {
+        stage('Build & publish packages') {
```

Comment:
@ajoubertza I've decided to merge building packages and triggering their publish into one stage. There's no point building packages if they're not going to be published. This will save time on resources on all non-master branch builds.

Any issues you foresee with this?

Reply:
Is it possible for the tests to pass, but a build to fail?  Then we might want test builds before merging.  I don't recall ever seeing a build of one of our packages failing (other than katgui, but that's not Python).

Reply:
True, any exception (non-zero exit) can cause a build to fail. However simply having the tests pass as the criteria for merging to `master` is a reasonable quality gate. 

The package step might still fail afterward (but this will still halt the release pipeline).